configure-proxy-formula/configure-proxy/config-answers.txt

```diff
@@ -1,18 +1,18 @@
 RHN_PARENT={{grains['master']}}
-HTTP_PROXY=''
-TRACEBACK_EMAIL=''
-USE_SSL=Y/n
-USE_EXISTING_CERTS=y/N
-INSTALL_MONITORING=n
-SSL_ORG=SUSE
-SSL_ORGUNIT=SUSE
+HTTP_PROXY={{ pillar.get('http_proxy') }}
+TRACEBACK_EMAIL={{ pillar.get('traceback_email') }}
+USE_SSL={% if {{ pillar.get('use_ssl') }} == true %} Y/n {% else %} y/N {% endif %}
```

Comment:
This should be hardcoded to 'Y' and new proxy setup do not have this option anymore, see https://github.com/uyuni-project/uyuni/commit/15d4228012ee3888142979f07dd2aa163e5734e1